src/calculateRentalCost.js

```diff
@@ -5,6 +5,25 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const shortTime = 2;
+  const avarageTime = 3;
+  const longTime = 7;
+  const costOfDay = 40;
+  const avarageTimeDiscount = 20;
+  const longTimeDiscount = 50;
+  const summ = days * costOfDay;
+
+  if (days >= longTime) {
+    return summ - longTimeDiscount;
+  }
+
+  if (days >= avarageTime) {
+    return summ - avarageTimeDiscount;
+  }
+
+  if (days <= shortTime) {
+    return summ;
+  }
```

Comment:
You don't need to check if 'days' is less than or equal to 'shortTime' at the end of the function. If none of the previous conditions are met, you can simply return 'summ'. This is because in JavaScript, if none of the previous 'if' conditions are met, the code execution will continue to the next lines after the 'if' statements.